_gettingstarted/requests.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: article
+title: requests
+description: Python's requests library is a powerful and user-friendly tool for making HTTP requests
```

Comment:
description can be more about the article, than the library. Something like "A beginner's guide to Python's requests library, a powerful and user friendly tool for making HTTP requests" could be helpful.